src/website/apps/api/v2/serializers/team.py

```
@@ -38,18 +61,42 @@
+        if instance.slug:
+            data.pop('id', None)
+        return data
+
     class Meta:
         model = Member
         fields = [
             'id', 'name', 'title', 'about', 'picture', 'picture_url', 'twitter',
-            'linked_in', 'order', 'descriptions', 'created', 'modified', 'is_deleted'
+            'linked_in', 'order', 'descriptions', 'created', 'modified', 'is_deleted',
+            'public_identifier', 'api_url', 'has_slug'
         ]
```

Ruff (0.12.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)